loadbalancer/lb_app_profile.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * NSX API
+ *
+ * VMware NSX REST API
+ *
+ * API version: 1.0.0
```

Comment:
Are we still on version 1.0.0 here?

Reply:
I think so, looks like it refers to API version (/api/v1/)